documentation/FlowManager.md

```diff
@@ -1,9 +1,12 @@
 ## FlowManager
 
-The flowmanager is the component in the system that is largely responsible for managing the flow. For example, the steps involved in logging in requires retrieving preferences, solutions, device data, etc. and passing this to the [MatchMaker Framework](MatchMakerFramework.md). And following that sending this via the [Context Manager](ContextManager.md) and then to the [Lifecycle Manager](LifecycleManager.md).
+The flowmanager is central point of coordination in the system for managing flow between different architecture components. 
+For example, it coordinates the steps involved in logging in requires retrieving preferences, solutions, device data, etc. and passing this to the [MatchMaker Framework](MatchMakerFramework.md). 
```

Comment:
This sentence needs a little fixing up: "it coordinates the steps involved in logging in requires retrieving preferences".